telemetry/definitions/commonDefinitions.json

```
@@ -5959,6 +5964,28 @@
                 }
             ]
         },
+        {
+            "name": "toolkit_componentInit",
```

most metrics are named with the "verb" first. also, `toolkit_init` exists so naming this similarly helps discoverability.

```suggestion
            "name": "toolkit_initComponent",
```

---

renamed to "toolkit_initModule" with its accompanying "module" metadata